source/base/Profiler.ooc

```diff
@@ -65,6 +70,8 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
+		while (This _profilers count)
```

Comment:
Is this OK? I remember when we discussed if it was OK to use NULL as a bool and decided it wasn't, i.e. `if (pointer == null)` instead of `if (pointer)`. I don't have a problem with it but someone might.